code/datums/components/crafting/recipes.dm

```diff
@@ -142,6 +142,142 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_WEAPON
 
+/datum/crafting_recipe/advancedegun
+	name = "Advanced Energy Gun"
+	tools = list(TOOL_SCREWDRIVER, TOOL_MULTITOOL)
```

Comment:
multitool see,s weird, its a tool security doesnt have by default and we shouldnt expect them to grab one, why not crowbar/wrench? they are in the wardens office and easier to get

Reply:
I swear security's toolbox comes with a multitool instead of a analyzer on at least metastation. And I could swear the other stations have one too either in the same toolbox or just lying out. Either in the warden's office or in the brig itself. Though I could be wrong.

Reply:
i dont think so?

Reply:
mappers dont have any ability to change what gets created in storage items and i dont think thres a sec subtype of any toolbox

Reply:
I'm gonna drop this down to just screwdriver and wirecutters for now, just because I think you're right about the multitools being a dubious feature of security.